spec/factories/application_form.rb

```diff
@@ -225,7 +225,7 @@
                        end
 
         hesa_sex = if sex == 'Prefer not to say'
-                     nil
+                     '96'
```

Comment:
This feels very "magic number"y do we have an object that knows the `key`, `title` and `hesa_code` for "Prefer not to say"?

Reply:
Well yes and no.  

I originally tried to clean this method up by just grabbing random data from the known hesa data. However, because there are places in our specs where we rewind back to 2020, when 'prefer not to say' gives us a different result than dates greater than 2021, and places where we are expecting very specific results (eg, when disabilities are not random, there are specs that are looking for the first two in the list this generates), many many tests fail. And each of them needs to be considered separately. Ie, why are we rewinding back so far? Why are we looking for these specific results? 

Changing this to 96 means that at least everything that is generated by the factory will have a hesa code, so things that are just expecting complete e&d data will pass. There are still areas where it isn't actually generating _valid_ data, because it is drawing on data that we no longer use to generate the models. 

so '96' is better than nil. But both are bad. The badness is going to take a separate and concerted effort to properly unpick.

Now you understand why this has taken me a week to do.

Reply:
🤯 actual footage of my brain right now